cordova-lib/src/configparser/ConfigParser.js

```diff
@@ -279,113 +269,108 @@ ConfigParser.prototype = {
 
         return scriptElements.filter(filterScriptByHookType);
     },
-
+   /**
+    * Returns a list of plugin (IDs)
+    * @return {string[]} Array of plugin IDs
+    */
+    getPluginIdList: function () {
+        var plugins = this.doc.findall('plugin');
+        var result = plugins.map(function(plugin){
+            return plugin.attrib.name;
+        });
+        var features = this.doc.findall('feature');
```

Comment:
A comment here to let future code readers would be great !
I can foresee people scratching their head as to if/why we are supporting both syntax.